src/utils/extraActionCreators/downloadProcessedMatrix.js

```diff
@@ -23,7 +22,6 @@ const downloadProcessedMatrix = (experimentId) => async (dispatch, getState) =>
 
     const body = {
       name: taskName,
-      embeddingETag: await getEmbeddingETag(experimentId, getState, dispatch),
```

Comment:
I am pretty sure this is encessary for the same reason as in trajectory. Check because I think this might be broken right now

Reply:
don in the api